packages/select/src/select.vue

```diff
@@ -366,10 +371,27 @@ export default defineComponent({
         if (reference.value.$el) {
           inputWidth.value = reference.value.$el.getBoundingClientRect().width
         }
+        if(ctx.slots.prefix){
+          const inputChildNodes = reference.value.$el.childNodes
+          const input = [].filter.call(inputChildNodes, item => item.tagName === 'INPUT')[0]
+          watch(() => ctx.slots.prefix(), () => {
+            nextTick(() => {
+              const prefix = reference.value.$el.querySelector('.el-input__prefix')
+              prefixWidth.value = Math.max(prefix.getBoundingClientRect().width + 5, 30)
+              if(states.prefixWidth){
+                input.style.paddingLeft = `${Math.max(states.prefixWidth, 30)}px`
+              }
+            })
```

Comment:
I do not think calling `$slot.xxx()` outside render function is a good idea, which could cause `Vue` warning. Can we have some work around on this?